doc/manual/manual.tex

```diff
@@ -1572,6 +1572,145 @@ \subsubsection{Free surface stabilization}
+
+\begin{align}
+  \label{eq:stokes-1-melt}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u_s)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u_s)\mathbf 1\right)
+                \right] + \nabla p_f + \nabla p_c  &=
+  \rho \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-2-melt}
+  \nabla \cdot \mathbf u_s - \nabla \cdot K_D \nabla p_f 
+  - K_D \nabla p_f \cdot \frac{\nabla \rho_f}{\rho_f}
+  &= 
+  - \nabla \cdot K_D \rho_f \mathbf g
+  \notag
+  \\
+  &\quad
+  + \Gamma \left( \frac{1}{\rho_f} - \frac{1}{\rho_s} \right)
+  \\
+  &\quad
+  - \frac{\phi }{\rho_f} \mathbf u_s \cdot \nabla\rho_f 
+  - \frac{1 - \phi }{\rho_s} \mathbf u_s \cdot \nabla\rho_s
+  \notag
+  \\
+  &\quad
+  - K_D \mathbf g \cdot \nabla \rho_f 
+  & \qquad
+  & \textrm{in $\Omega$},
+  \notag
+  \\
+  \label{eq:stokes-3-melt}
+  \nabla \cdot \mathbf u_s + \frac{p_c}{\xi} 
+  &=
+  0.
+\end{align}
+
+We use the indices $s$ to indicate properties of the solid and $f$ for the properties of the fluid. 
+The equations are solved for the solid velocity $\mathbf{u_s}$, the fluid pressure $p_f$, and an additional 
+variable, the compaction pressure $p_c$, which is related to the fluid and solid pressure through the relation 
+$p_c = (1-\phi) (p_s-p_f)$. $K_D$ is the Darcy coefficient, which is defined as the quotient of the permeability 
+and the fluid viscosity and $\Gamma$ is the melting rate. $\eta$ and $\xi$ are the shear and compaction viscosities 
+and can depend on the porosity, temperature, pressure, strain rate and composition. However, there are various 
+laws for these quantities and so they are implemented in the material model. Common formulations for the dependence 
+on porosity are $\eta = (1-\phi) \eta_0 e^{-\alpha_\phi \phi}$ with $\alpha_\phi \approx 25...30$ and 
+$\xi = \eta_0 \phi^{-n}$ with $n \approx 1$.
+
+To avoid the density gradients in Equation \ref{eq:stokes-2-melt}, which would have to be specified individually 
+for each material model by the user, we can use the same method as for the mass conservation (described in section 
+\ref{sec:boussinesq}) and assume the change in solid density is dominated by the change in static pressure, 
+which can be written as
+$\nabla p_s \approx \nabla p_\text{static} \approx \rho_s \textbf{g}$.
```

Comment:
When we're in CA, I'll have you explain to me why this is a valid approximation here :-)